runtime/runtime-params-estimator/test-contract/build.sh

```diff
@@ -29,6 +29,12 @@ echo ${bare_wasm}
 # bloat its size to the given values.
 
 # 10KiB
+
+# hotfix for people building cargo with debug flags, causing binary to be 35kb
```

Comment:
Shouldn't users always build release when testing performance?

Reply:
I'm running release build with debug symbols on, so the performance shouldn't be affected. It's useful for debugging though.

We need files produced by this build script for running `cargo test`. 
That wasn't the case in the past. 
By default all tests run when you execute `cargo test`, you have no control over it. So this is a regression.

Reply:
I would also be fine if we don't merge `build.sh` fix, but instead disable broken tests mentioned in https://github.com/near/near-ops/issues/440
Alternatively, we can commit 
```

49 | |     CODE_10K => "test-contract/res/stable_small_contract.wasm",
50 | |     CODE_100K => "test-contract/res/stable_medium_contract.wasm",
51 | |     CODE_1M => "test-contract/res/stable_large_contract.wasm",
``` to the repo

Reply:
>I'm running release build with debug symbols on, so the performance shouldn't be affected. It's useful for debugging though.

I think there's a bit of confusion here. There are two "debug" related cargo toml fields. `debug` controls debug symbols (`-g`). `debug-assertions` controls `debug_assert`. The `lazy_static_include` crate uses the latter to switch between compile time and runtime inclusion.

Reply:
https://github.com/near/nearcore/pull/4125 fixes the compilation error.

Reply:
I'll close this PR then.